test/load/locust-files/fleet_autoscaling.py

```diff
@@ -34,7 +34,7 @@ def scaleUpFleet(self):
             "apiVersion": "agones.dev/v1",
             "kind": "Fleet",
             "metadata": {
-                "generateName": "fleet-simple-udp",
+                "generateName": "simple-game-server",
```

Comment:
simple-game-server-fleet?

Reply:
Sure. :shrug: I don't think anyone is using this anyway.

Reply:
Switched to "fleet-simple-game-server" since the previous version started with "fleet"

Reply:
Since this is in the locust dir, I agree that it isn't being used actively (the old udp-server version was 0.5 whereas everywhere else it was 0.21). 

Not a big deal, but it doesn't look like you actually pushed a change with the updated fleet name.

Reply:
huh, I changed it here:
https://github.com/googleforgames/agones/pull/1992/commits/3152f31dbc2c2df9b9d14598ece8a202b19d1c87#diff-29dc460d65f66644f6c926f4f634a479e9167383eaeb8c8078f76090cd391fcbR174

Did I do the wrong file... :thinking: ?

Reply:
OIC THE SPOT! Pushing up a fix.